CentralDeErros.Data/Maps/OccurrenceMap.cs

```diff
@@ -13,13 +13,11 @@ public class OccurrenceMap : IEntityTypeConfiguration<Occurrence>
     {
         public void Configure(EntityTypeBuilder<Occurrence> builder)
         {
-
             builder
-                .HasKey(k => k.Id);
+                .ToTable("occurrence");
```

Comment:
Aqui é mais uma questão estilística, mas se o padrão for esse para a título nas tabelas, posso seguir 👍

Reply:
Um comentário geral sobre essas propostas de alterações, é que são válidas, mas tem que pensar que os dois modelos estão funcionais. Então, não necessariamente é preciso mudar. Acho que vai mais de uma opção de desenvolvimento do que uma questão de se funciona ou não.

Reply:
Neste ponto seria interessante seguimos o padrão de declarar no nome da table, não sei se seria certo não declara-lo.